applications/MeshMovingApplication/tests/mesh_moving_test_case.py

```diff
@@ -82,7 +82,7 @@ def __SetPrintOfReferenceResults(self):
                                           "MESH_VELOCITY_X",
                                           "MESH_VELOCITY_Y",
                                           "MESH_VELOCITY_Z"],
-                    "output_file_name" : \""""+result_file_name+"""\",
+                    "output_file_name" : \""""+result_file_name.replace("\\", "\\\\")+"""\",
```

Comment:
ouch sorry guys this is my bad, I didn't know how to do this in a portable way :/

Would you say this is a good solution?

Reply:
@philbucher Is not your fault, is windows.
In the script the path is constructed correctly using `os.path.join`, which adds a `\` as a separator, but that only works for win, and since this is being processes by the json lib it crashes (because is trying to escape the char instead of reading the single backslash).

The problem is that, if you use `/` then when windows tries to open the file it doesn't exists. 

To be honest I can't think of any other solution right know, but if you find one please feel free to change it :)